.setup/bin/sample_courses/models/course/course_generate_utils.py

```diff
@@ -172,31 +191,37 @@ def make_course_json(self) -> None:
         gradeables_json_output["benchmark_percent"] = {}
         for i in range(len(benchmarks)):
             gradeables_json_output["display_benchmark"].append("lowest_" + benchmarks[i])
-            gradeables_json_output["benchmark_percent"]["lowest_" + benchmarks[i]] = 0.9 - (0.1 * i)
+            gradeables_json_output["benchmark_percent"]["lowest_" + benchmarks[i]] = \
+                0.9 - (0.1 * i)
 
         gradeables_json_output["section"] = section_ta_mapping
         messages = [f"<b>{course_id} Course</b>",
-                    "Note: Please be patient with data entry/grade corrections for the most recent "
-                    "lab, homework, and test.",
-                    "Please contact your graduate lab TA if a grade remains missing or incorrect for more than a week."]
+                    "Note: Please be patient with data entry/grade corrections for the most "
+                    "recent lab, homework, and test.", "Please contact your graduate lab TA "
+                    "if a grade remains missing or incorrect for more than a week."]
         gradeables_json_output["messages"] = messages
 
         # Attempt to write the customization.json file
         try:
-            with open(os.path.join(customization_path, "customization_" + course_id + ".json"), 'w') as customization_file:
+            build_customization_path = os.path.join(customization_path,
+                                                    "customization_" + course_id + ".json")
+            with open(build_customization_path, 'w') as customization_file:
                 customization_file.write("/*\n"
-                                         "This JSON is based on the automatically generated customization for\n"
-                                         f"the development course \"{course_id}\" as of {NOW.strftime('%Y-%m-%d %H:%M:%S%z')}.\n"
-                                         "It is intended as a simple example, with additional documentation online.\n"
+                                         "This JSON is based on the automatically "
+                                         "generated customization for\n"
+                                         f"the development course \"{course_id}\" as of "
+                                         f"{NOW.strftime('%Y-%m-%d %H:%M:%S%z')}.\n"
+                                         "It is intended as a simple example, "
+                                         "with additional documentation online.\n"
                                          "*/\n")
             json.dump(gradeables_json_output,
-                      open(os.path.join(customization_path, "customization_" + course_id + ".json"), 'a'),indent=2)
+                      open(os.path.join(customization_path,
+                                        "customization_" + course_id + ".json"), 'a'), indent=2)
```

Comment:
You could use a temporary variable for the path rather than breaking nested function calls into multiple lines.